mdx-models/src/main/java/com/mx/path/model/mdx/model/id/v20240209/Authentication20240209.java

```diff
@@ -0,0 +1,55 @@
+package com.mx.path.model.mdx.model.id.v20240209;
+
+import java.util.List;
+
+import lombok.Data;
+
+import com.google.gson.annotations.SerializedName;
+import com.mx.path.model.mdx.model.MdxBase;
+import com.mx.path.model.mdx.model.challenges.Challenge;
+
+@Data
+public class Authentication20240209 extends MdxBase<Authentication20240209> {
+  private final String version = "20240209";
```

Comment:
Is there any reason not to make this `public static`?

Reply:
No. Just prototyping. 😆